tests/test_partial.py

```diff
@@ -32,7 +32,7 @@ def module_venv(pytestconfig) -> abc.Iterator[env.VirtualEnv]:
 
 @pytest.fixture(scope="function")
 def module_venv_active(
-    deactive_venv: None,
+    deactive_venv: abc.Iterator[None],
```

Comment:
I don't think this is correct. The return type of the fixture is `abc.Iterator[None]`, but the value of the argument here is `None`, isn't it?